back end/main.py

```diff
@@ -62,7 +65,7 @@ def main(word, language, part_of_speech):
     sentences_with_wordPOS = vocabulary.get_sentences_with_part_of_speech(word, part_of_speech, sentences)
     if sentences_with_wordPOS == []:
         return [['No sentences found']]
```

Comment:
We also have to print this.

Reply:
Printig it causes the front end to crush. With return of it, front end gets the result and prints it the right way to the UI